### PR TITLE
feat: Add support for multiple consecutive banners in configs.

### PR DESCRIPTION
**Pull request by swanandtambe**

### CODE REVIEW

File: tests/unit/mock/config/parser/base/cisco_iosxr/iosxr_multiple_banners_sent.txt [hunk 1]
```diff
@@ -0,0 +1,18 @@
+hostname multi-banner-xr
+!
+banner exec ~
+* exec banner line 1 *
```
Comment (itdependsnetworks) [inline tests/unit/mock/config/parser/base/cisco_iosxr/iosxr_multiple_banners_sent.txt:4]: can you add a test that shows an empty line here?

Comment (swanandtambe) [reply tests/unit/mock/config/parser/base/cisco_iosxr/iosxr_multiple_banners_sent.txt:4]: Is this to test an empty (whitespace only) line in the banner?

Comment (itdependsnetworks) [reply tests/unit/mock/config/parser/base/cisco_iosxr/iosxr_multiple_banners_sent.txt:4]: not really, something like this 

```
banner exec ~
* exec banner line 1 

* exec banner line 2
```
It was not clear to me from the code if this was taken into consideration.

**Review by jeffkala — APPROVED** [review]
